stix_shifter/stix_transmission/src/modules/aws_cloud_watch_logs/aws_cloud_watch_logs_boto3_client.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+import string
+import random
+
+
+class BOTO3Client:
+    def __init__(self, connection, configuration):
+        host = connection.get('host')
+        region_name = host.split('.')[1]
```

Comment:
You dont need to get the regional endpoint from the user. Ideally you should just get the region name and what service in this case CloudwatchLogs(`logs`) and construct the endpoint/host. 
As you are using the SDK, you dont even need to construct the endpoint because sdk handles that itself when you provide the service name. In my opinion it should be-
```
host = connection.get('host') //default value can be 'logs.amazonaws.com' in the ui
// region_name stays inside options attribute of connection object
region_name = connection.get('options',{}).get("region_name", None) 
```

Reply:
We will look into it.

Reply:
done